docs/source/release/v6.6.0/indirect_geometry.rst

```diff
@@ -7,23 +7,25 @@ Indirect Geometry Changes
 
 New Features
 ------------
-.. amalgamate:: Indirect/New_features
+- A new interface has been created: the Indirect Data Manipulation Interface, from the Indirect Data Analysis interface the Sqw, Moments, and Symmetrise tabs have been moved into the new interface along with Elwin and Iqt from the Indirect Data Analysis interface.
+- Loads data from the beam monitor and  adds missing environment parameters, reactor power and scan period to the log when loading EMU event files using :ref:`LoadEMU <algm-LoadEMU>`.
```

Comment:
There's an extra space in this sentence between "beam monitor and" and "adds missing environment"